cdc/sinkv2/eventsink/mq/worker.go

```diff
@@ -289,6 +298,48 @@ func (w *worker) asyncSend(
 	return nil
 }
 
+func (w *worker) sendMessages(ctx context.Context) error {
+	inputCh := w.encoderGroup.Output()
+	ticker := time.NewTicker(15 * time.Second)
+	metric := codec.EncoderGroupResponseChanSizeGauge.
+		WithLabelValues(w.changeFeedID.Namespace, w.changeFeedID.ID)
+	defer func() {
+		ticker.Stop()
+		codec.EncoderGroupResponseChanSizeGauge.
+			DeleteLabelValues(w.changeFeedID.Namespace, w.changeFeedID.ID)
+	}()
+	for {
+		select {
+		case <-ctx.Done():
+			return errors.Trace(ctx.Err())
+		case <-ticker.C:
+			metric.Set(float64(len(inputCh)))
+		case future, ok := <-inputCh:
```

Comment:
Will batch get event from this channel bring better performance?

Reply:
batch can improve performance by reducing frequent function calls to avoid the cost of context switch.

If we want to batch multiple futures here to improve performance, the `AsyncSendMessage` method should also be changed to take a batch a messages as parameters.